src/ethereum_test_forks/base_fork.py

```diff
@@ -462,6 +462,18 @@ def evm_code_types(cls, block_number: int = 0, timestamp: int = 0) -> List[EVMCo
         """Return list of EVM code types supported by the fork."""
         pass
 
+    @classmethod
+    @abstractmethod
+    def max_code_size(cls) -> int | None:
```

Comment:
Should we also introduce an `account_load_cost_calc` to provide a way to dynamically calculate the costs of loading an account depending on the fork and code size?